libs/qa/dataset_summary.py

```diff
@@ -18,35 +15,22 @@
     CommonFields.COUNTRY,
     CommonFields.COUNTY,
     CommonFields.AGGREGATE_LEVEL,
-    CommonFields.LOCATION_ID,
+    CommonFields.FIPS,
 ]
 
 VARIABLE_FIELD = "variable"
 
 SUMMARY_PATH = dataset_utils.DATA_DIRECTORY / "timeseries_summary.csv"
 
 
-class TimeseriesSummary(pydantic.BaseModel):
```

Comment:
Should we just remove the dataset summary? Don't really think it's used anymore and we could always bring it back if needed

Reply:
Done! Please take another look.